packages/react-data-grid/src/common/editors/__tests__/EditorContainer.spec.tsx

```diff
@@ -106,60 +102,61 @@ describe('Editor Container Tests', () => {
       }
     }
 
-    beforeEach(() => {
-      ({ wrapper, props } = setup({
+    function innerSetup() {
+      return setup({
         value: 'SupernaviX',
         column: { ...fakeColumn, editor: <TestEditor /> }
-      }));
-    });
+      });
+    }
 
     it('should render element custom editors', () => {
+      const { wrapper } = innerSetup();
       const editor = wrapper.find(TestEditor);
 
       expect(editor.length).toBe(1);
-      expect(editor.props().value).toBe('SupernaviX');
-      expect(editor.props().onCommit).toBeDefined();
-      expect(editor.props().onCommitCancel).toBeDefined();
+      expect(editor.prop('value')).toBe('SupernaviX');
+      expect(editor.prop('onCommit')).toBeDefined();
+      expect(editor.prop('onCommitCancel')).toBeDefined();
     });
 
     it('should not commit if any element inside the editor is clicked', () => {
+      const { wrapper, props } = innerSetup();
       wrapper.find('#input1').simulate('click');
       wrapper.find('#input2').simulate('click');
 
-      expect(props.onCommit.mock.calls.length).toEqual(0);
+      expect(props.onCommit).not.toHaveBeenCalled();
     });
 
     it('should not commit if any element inside the editor is clicked that stops the event propagation', () => {
+      const { wrapper, props } = innerSetup();
       wrapper.find('#button1').simulate('click');
       wrapper.find('#button2').simulate('click');
 
-      expect(props.onCommit.mock.calls.length).toEqual(0);
+      expect(props.onCommit).not.toHaveBeenCalled();
     });
 
     it('should call onCommitCancel when editor cancels editing', () => {
+      const { wrapper, props } = innerSetup();
       const editor = wrapper.find(TestEditor);
```

Comment:
I remember `find` has a generic type like 
```
wrapper.find<TestEditorProps (or just TestEditor, I forgot it exactly)>(TestEditor)
```
then we can avoid the cast next line

Reply:
Fixed.